orttraining/orttraining/test/python/onnxruntime_test_postprocess.py

```diff
@@ -0,0 +1,240 @@
+                           None,
+                           model_desc,
+                           "LambOptimizer",
+                           map_optimizer_attributes,
+                           lr_desc,
+                           device,
+                           world_rank=0,
+                           world_size=1,
+                           _opset_version=12)
+
+        train_output = model.train_step(*inputs)
+        return model.onnx_model_
+
+    def count_all_nodes(self, model):
+        return len(model.graph.node)
+    
+    def count_nodes(self, model, node_type):
+        count = 0
+        for node in model.graph.node:
+            if node.op_type == node_type:
+                count += 1
+        return count
+
+    def find_nodes(self, model, node_type):
+        nodes = []
+        for node in model.graph.node:
+            if node.op_type == node_type:
+                nodes.append(node)
+        return nodes
+
+    def get_name(self, name):
+        if os.path.exists(name):
+            return name
+        rel = os.path.join("testdata", name)
+        if os.path.exists(rel):
+            return rel
+        this = os.path.dirname(__file__)
+        data = os.path.join(this, "..", "..", "..", "..", "onnxruntime", "test", "testdata")
+        res = os.path.join(data, name)
+        if os.path.exists(res):
+            return res
+        raise FileNotFoundError("Unable to find '{0}' or '{1}' or '{2}'".format(name, rel, res))
+
+    def test_layer_norm(self):
+        class LayerNormNet(nn.Module):
+            def __init__(self, target):
+                super(LayerNormNet, self).__init__()
+                self.ln_1 = nn.LayerNorm(10)
+                self.loss = nn.CrossEntropyLoss()
+                self.target = target
+
+            def forward(self, x):
+                output1 = self.ln_1(x)
+                loss = self.loss(output1, self.target)
+                return loss, output1
+
+        device = torch.device("cpu")
+        target = torch.ones(20, 10, 10, dtype=torch.int64).to(device)
+        model = LayerNormNet(target)
+        input = torch.randn(20, 5, 10, 10, dtype=torch.float32).to(device)
+
+        input_desc = IODescription('input', [], "float32")
+        output0_desc = IODescription('output0', [], "float32")
+        output1_desc = IODescription('output1', [20, 5, 10, 10], "float32")
+        output2_desc = IODescription('output2', [20, 5, 10, 10], "float32")
+        model_desc = ModelDescription([input_desc], [output0_desc, output1_desc])
+
+        learning_rate = torch.tensor([1.0000000e+00]).to(device)
+        input_args=[input, learning_rate]
+
+        onnx_model = self.get_onnx_model(model, model_desc, input_args, device)
+
+        count_layer_norm = self.count_nodes(onnx_model, "LayerNormalization")
+        count_nodes = self.count_all_nodes(onnx_model)
+       
+        assert count_layer_norm == 1
+        assert count_nodes == 3
+
+    def test_expand(self):
```

Comment:
In this test I the learning rate as the last input in input_args with the descriptor " IODescription('Learning_Rate', [1,], torch.float32)". But I still get a binding 'Learning_Rate' error. I follow the same logic in test_layer_norm an don't get the error. Any idea why this is happening?